src/transactions/scriptComposer/index.ts

```diff
@@ -58,7 +58,9 @@ export class AptosScriptComposer {
 
     // Load the calling type arguments into the loader.
     if (input.typeArguments !== undefined) {
-      await Promise.all(input.typeArguments.map((typeTag) => this.builder.load_type_tag(nodeUrl, typeTag.toString())));
+      for (const ty of input.typeArguments) {
```

Comment:
nit: for readability
```suggestion
      for (const typeArgument of input.typeArguments) {
```